kernel/src/main/java/org/cristalise/kernel/lifecycle/instance/predefined/agent/SetAgentPassword.java

```diff
@@ -56,7 +56,7 @@ protected String runActivityLogic(AgentPath agent, ItemPath item, int transition
             AgentPath targetAgent = new AgentPath(item);
             String newPwd;
 
-            if (!targetAgent.equals(agent) && !agent.hasRole("Admin"))
+            if (!targetAgent.equals(agent) && !agent.hasRole(ADMIN_ROLE.getName()))

```

Comment:
Create if this code can be replaced by Permissions

Reply:
This request is tracked here: https://github.com/cristal-ise/cristal-ise/issues/211